nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ConsumeEWS.java

```diff
@@ -107,6 +109,14 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .sensitive(true)
             .build();
+    public static final PropertyDescriptor MAILBOX = new PropertyDescriptor.Builder()
+            .name("mailbox")
+            .displayName("Mailbox")
+            .description("Mailbox")
+            .required(false)
```

Comment:
I would add a default value just to make sure querying this property at least returns an empty string:
`.defaultValue("")` is used in other properties for this reason already.